modules/websocket-client/src/index.ts

```diff
@@ -0,0 +1,209 @@
+import WebSocket from "./websocket";
+import { randomStringWithTimeStamp } from "@phenyl/utils";
+import {
+  GeneralRequestData,
+  GeneralResponseData,
+  SubscriptionRequest,
+  SubscriptionResult,
+  VersionDiff,
+  VersionDiffListener,
+  VersionDiffSubscriber,
+  WebSocketServerSubscriptionResultMessage,
+  WebSocketServerVersionDiffMessage,
+  WebSocketServerResponseDataMessage
+} from "@phenyl/interfaces";
+
+type Id = string;
```

Comment:
I defined Id type in websocket-client because interfaces doesn't export Id type.
Tell me if there is a better way.

Reply:
This should be ok for now. We can change this when we add `Id` type to `@phenyl/interfaces`